features/page-setup.yml

```diff
@@ -0,0 +1,13 @@
+name: Page setup
+description: The `@page` rule sets the page-specific dimensions, orientation, and margins for content such as printed documents, ebooks, or slides.
+spec: https://drafts.csswg.org/css-page-3/#at-page-rule
+group:
+  - layout
+  - print
+caniuse: css-paged-media
+compat_features:
+  - css.at-rules.page
+  - css.at-rules.page.page-orientation
```

Comment:
I think I'd recommend moving `css.at-rules.page.page-orientation` into its own feature, and then pinning baseline on `css.at-rules.page.size`.

It looks like support for @page is incorrect in Safari. I opened https://github.com/mdn/browser-compat-data/issues/24566, and I think that will block this.

Reply:
I sent https://github.com/mdn/browser-compat-data/pull/24583 to fix this.